src/modules/vendor/components/programs/cards/templates/QuestionAnswer.vue

```diff
@@ -94,14 +101,31 @@ export default {
         NotifyNegative('Erreur lors de l\'ajout de la réponse.');
       }
     },
+    async deleteQuestionAnswer (index) {
+      try {
+        const answerId = get(this.card, `questionAnswers[${index}]`)._id;
+        await Cards.deleteAnswer({ cardId: this.card._id, answerId });
```

Comment:
Le get de lodash permet de ne pas avoir d'erreur si un champ n'existe pas. Du coup ici si questionAnswers[index] n'existe pas ca renverra null (ou undefined je ne sais plus) et comme tu essayes de prendre l'_id de ce resultat ca va renvoyer une erreur. 
Du coup il faudrait que tu fasses plutot `get(this.card, `questionAnswers[${index}]._id`)`

Tu peux aussi ajouter `if (!answerId) return;` pour eviter un appel au back inutile

Reply:
daccord merci bcp pour lexplication